docs/compilers/Visual Basic/Compiler Breaking Changes - VS2015.md

```diff
@@ -108,3 +108,15 @@ According to @AlekseyTs , "Dev10 considers some expressions as invalid in contex
 
 These rules are being relaxed in Roslyn.  A constant expression will be allowed of the form expr.ID where ID is a constant member of the type of the expression.  The expression will not be evaluated and a warning will be issued.  There are no restrictions on the form of expr, other than that it must be syntactically and semantically valid.
 
+### VB Embedded Runtime inherits overflow checking from the compilation
+
+See https://github.com/dotnet/roslyn/issues/6941. Some VB runtime methods are specified to throw an `OverflowException` when a converted value overflows the target type. When compiling a VB project with the runtime embedded (`/vbruntime`), the compiler includes the necessary VB runtime helpers into the assembly that is produced.  These runtime helpers inherit the overflow checking behavior of the VB.NET project that they are embedded into.  As a result, if you both embed the runtime and have overflow checking disabled (`/removeintchecks`), you will not get the specified exceptions from the runtime helpers.  Although technically it is a bug (Visual Studio 2013 did not have this bug), we have found that customers would be broken by having it fixed, so we do not expect to change this behavior.
```

Comment:
>Visual Studio 2013 did not have this bug [](start = 649, length = 40)

"Visual Studio 2013 did not have this bug" Is this accurate? I see the same behavior with native compiler and comments in the issue indicate that there is no behavior change in Roslyn